### ISSUE TITLE
Take a closer look at the UK digital design system

### ISSUE THREAD

**Issue by froi** [Code4PuertoRico/code4pr-design#3]
Gov UK son los pioneros en lo que llamamos digital services en el gov. Seria bueno darle una buena mirada a su sistema de diseño https://design-system.service.gov.uk/

**Comment by froi** [comment]
@LaysCurbelo ya que tu me hiciste llegar esto me gustaria tus comentarios. 😄

**Comment by pablodefendini** [comment]
El del gobierno de EEUU es muy bueno también—lo desarrollaron en 18F si mal no recuerdo, y la mayoría de los componentes están en el public domain (y los que no están detrás de la licencia de MIT y similar): https://github.com/uswds/uswds

**Comment by froi** [comment]
> El del gobierno de EEUU es muy bueno también—lo desarrollaron en 18F si mal no recuerdo, y la mayoría de los componentes están en el public domain (y los que no están detrás de la licencia de MIT y similar): https://github.com/uswds/uswds

Yep, fue una colaboración entre 18F/TTS, USDS y otros. El progreso a sido super bueno. El [US Forms System](https://github.com/usds/us-forms-system) tambien sale de esfuerzos similares.
